### ISSUE TITLE
The icon of 'Refresh' displays unfriendly in Dark/HC-Black

### ISSUE THREAD

**Issue by v-dantch** [Microsoft/AzureStorageExplorer#326]
**Storage Explorer Version**: 1.2.0
**Platform/OS Version**: Win10RS4
**Architecture**: ia32
**Build Number**: 20180619.1
**Commit**: 3b2233d5
**Regression From**: Not a Regression

#### Steps to Reproduce: ####
1. Change theme to Dark/HC-Black.
2. Select one storage account and check the icon of 'Refresh' icon from the 'Action' window.
3. Check whether the icon displays well or not.

#### Expected Experience: ####
The icons of 'Refresh' displays friendly.

#### Actual Experience: ####
The icons of 'Refresh' displays unfriendly.
![compare](https://user-images.githubusercontent.com/34729092/41647583-487da624-7466-11e8-893c-ee810872644f.PNG)

#### More Info: ####
This issue also reproduces for the icon of 'Add to Quick Access/ Delete'.

**Comment by craxal** [comment]
It looks perfectly fine to me. If you're referring to the white halo around the refresh icon, that's intended. If you look, the halo is present around the other icons.

**Comment by v-dantch** [comment]
Got it. Thanks for your update.
